src/Pages/ContentListTable/components/EditContentModal/EditContentForm.tsx

```diff
@@ -522,6 +523,26 @@ const EditContentForm = ({
                       setSelected={(value) => setVersionSelected(value, index)}
                     />
                   </FormGroup>
+                  <FormGroup fieldId='enable_module_hotfixes'>
+                    <Switch
+                      label='Modularity filtering disabled'
+                      labelOff='Modularity filtering enabled'
```

Comment:
Could you add an ouiId flag to this similar to what was done [here](https://github.com/content-services/content-sources-frontend/pull/197) for QE.

Reply:
added!